src/Form/ConfigForm.php

```diff
@@ -84,12 +84,23 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
 
     $form['access_token_ttl'] = [
       '#type' => 'number',
-      '#title' => $this->t('Access Token Expiration (in seconds)'),
+      '#title' => $this->t('Access Token Expiration'),
       '#default_value' => $cool_settings['access_token_ttl'] ?? 0,
+      '#field_suffix' => $this->t('seconds'),
       '#min' => 0,
       '#required' => TRUE,
     ];
 
+    $form['new_file_interval'] = [
+      '#type' => 'number',
+      '#title' => $this->t('New File interval'),
+      '#default_value' => $cool_settings['new_file_interval'] ?? 0,
+      '#field_suffix' => $this->t('seconds'),
+      '#min' => 0,
+      '#description' => $this->t('Minimum interval of time required to create a new File upon save. A value of 0 will not create any entity.'),
```

Comment:
"file" should be lowercase. But also I think we can improve this.
```suggestion
      '#description' => $this->t('Period during which a save operation will overwrite the existing file, rather than creating a new file.<br>
If the value is 0, the existing file will always be overwritten, and no new file will be created.<br>
This equally applies both to autosave, and to the editor\'s save button.'),
```
or
```suggestion
      '#description' => $this->t('Determines whether a save operation creates a new file entity.<br>
If the file entity currently attached to the media entity is newer than the interval specified here, the contents of the existing file are replaced.<br>
If the file entity is older than the interval specified here, a new file entity is created, while the old file entity remains unchanged.<br>
If the value is 0, the existing file will always be overwritten, and no new file will be created.'),
```
I was thinking "younger" but I think "newer" is better when talking about files.
(still not 100% sure)

Reply:
For the record: Now `0` has a different meaning here than it has in the access token ttl.
For the access token, 0 means "always renew", for new file interval 0 means "never create new".
But this is what we decided in chat, so let's keep it until we change our minds.

Reply:
For file and File, I use uppercase to refer to the entity name, since it contains the reference to the file I felt it was more apropiated.

Not sure where I got this.

Reply:
I prefer first option.